src/Our.Umbraco.Ditto/ComponentModel/TypeConverters/MultipleMediaPickerConverter.cs

```diff
@@ -0,0 +1,73 @@
+﻿namespace Our.Umbraco.Ditto
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using System.Globalization;
+    using System.Linq;
+
+    using global::Umbraco.Core.Models.PublishedContent;
+    using global::Umbraco.Web;
+
+    /// <summary>
+    /// Provides a unified way of converting multi media picker properties to strong typed collections.
+    /// </summary>
+    /// <typeparam name="T">
+    /// The <see cref="Type"/> of the node to return.
+    /// </typeparam>
+    public class MultipleMediaPickerConverter<T> : TypeConverter where T : PublishedContentModel
```

Comment:
Same here, do we need the `PublishedContentModel` constraint? (apologies for duplicate comments, just flagging them)